### DISCUSSION TITLE
Welcome to Dash-by-Plotly Discussions!

### DISCUSSION THREAD

**Discussion by Coding-with-Adam** [Coding-with-Adam/Dash-by-Plotly#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * We're better together: help other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do or will do with Dash Plotly.

**Comment by Hootie9lives** [comment]
Thanks for sharing your knowledge. I'm a data scientist, appreciate your work

**Reply by Coding-with-Adam** [reply]
You are most welcome, @Hootie9lives 
What do you plan to use Dash Plotly for? 

Adam,

**Reply by Hootie9lives** [reply]
We use it as a web application dev framework. Most of our data science workforce don't have the skills to dev in web native technologies (html, CSS, JavaScript), but everyone know Python.

**Reply by Hootie9lives** [reply]
also various graphing components are great as most don't know how to use D3

**Reply by Coding-with-Adam** [reply]
OK that's really good. I'm glad you're using Dash. If you or your team have any questions along the way, feel free to let me know.  

Kind regards,
Adam

**Reply by Hootie9lives** [reply]
Thank you Adam. Will do.

**Comment by Remus9744** [comment]
Hi Adam,

First of all, thanks for your Youtube Video and your code generating the map.
It's been very helpful and I'm trying to adapt it to my case.

Unfortunately, I've met some issue...
When I plot my map of "customers longitudes and lats", I get something that is "translated".
You'll see what I mean with the figures uploaded below =)

When I then set df.tail[-1] in the corresponding section, the points are plotted correclty on the map...
I am stuck there and I don't know how to solve this problem...

Do you have any idea that would solve this ?

Thanks for your help ! =)

![Map_Translated_2](https://user-images.githubusercontent.com/60917643/128366931-7d580ae1-8de7-4268-8fee-cb434d974a08.JPG)
![Map_Translated_1](https://user-images.githubusercontent.com/60917643/128366934-59f506b0-ec48-466b-b26d-ff98f5b9dfaa.JPG)
![Map_Translated_3](https://user-images.githubusercontent.com/60917643/128366935-d5cb1a0c-df7a-4986-aa43-bae91a86e675.JPG)
![Map_Translated_4](https://user-images.githubusercontent.com/60917643/128366945-e3ec306d-ac9d-40f6-adee-2074a4dc0b0d.JPG)

**Reply by DashBookProject** [reply]
HI @Remus9744 ,
I recommend you post this question/problem on Plotly Forum. This happened to me as well with certain datasets and I tried to look into it for a whole week but couldn't find a solution. I wonder if it's a bug in dataShader. Not sure... but I really had a hard time with this and could never fix one of my maps that I was going to use for a YouTube tutorial. 

Kind regards, 
Adam

**Comment by cARMO85** [comment]
thanks for the resource.
